kafka-connect-kudu/src/main/scala/com/datamountaineer/streamreactor/connect/kudu/sink/KuduWriter.scala

```diff
@@ -187,16 +187,23 @@ class KuduWriter(client: KuduClient, setting: KuduSettings) extends StrictLoggin
       val schema = record.valueSchema()
       val version = schema.version()
       val table = setting.topicTables(topic)
-      val cachedSchema = schemaCache.getOrElse(topic, SchemaMap(version, schema))
+
+      var currentKcql = setting.kcql.filter(r => r.getTarget.trim == table)
```

Comment:
@sdlcwangsong Why are these variables `var` not `val`?

Reply:
ok, I will modify var  to val